prove/Develop04/Program.cs

```diff
@@ -210,6 +146,13 @@ void ShowSpinnerWithText(string text, string position = "left")
                 case "4":
                     //---------------------------- User decide for finish the program -----------------------
                     quitProgramm = true;
+                    DateTime end = DateTime.Now;
+                    
+                    Console.Clear();
+                    Console.WriteLine($"Current session started at {start.ToString("HH:mm:ss")}");
+                    Console.WriteLine($"Current session end at {end.ToString("HH:mm:ss")}");
+                    Console.WriteLine($"Elapsed time: {elapsedTimeString}");
+                    Console.WriteLine();
```

Comment:
The final message with the information of time between start, end and time wasted in the app